test/loadtime/payload/payload.proto

```diff
@@ -9,9 +9,10 @@ import "google/protobuf/timestamp.proto";
 // encoded representation, making it ideal for the loadtime usecase which aims to
 // keep the generated transactions small.
 message Payload {
-  uint64 connections = 1;
-  uint64 rate = 2;
-  uint64 size = 3;
-  google.protobuf.Timestamp time = 4;
-  bytes padding = 5;
+  uint64                    connections = 1;
+  uint64                    rate        = 2;
+  uint64                    size        = 3;
+  google.protobuf.Timestamp time        = 4;
+  bytes                     id          = 5;
```

Comment:
I know I know, this could be after the padding, but nothing uses this yet and I have a sort of mild preference for having padding be at the end for now. Obviously if we add more fields we can and should just add values after the padding's 7 field tag.

Reply:
Seems correct to me to have the padding at the end, as you have it.

Reply:
How about giving the padding number "10" instead of "6", so we _could_ add a few more fields in front of it if needed?